### ISSUE TITLE
Does not work for UK phone numbers

### ISSUE THREAD

**Issue by Nosh118** [theOneAndOnlyOne/BeReel#5]
Attempting to login with a UK phone number in the format +44xxxxxxxxxx does not work, and instead returns the error below:

<img width="783" alt="image" src="https://github.com/theOneAndOnlyOne/BeReel/assets/67752804/222f34bc-2f52-44df-a0b9-191120ac8c65">

Works perfectly for a random US number.

**Comment by LinaAlf** [comment]
same here in switzerland, with +41, it does also not work sadly

**Comment by BingoDeeVee** [comment]
Also getting the same. 

*Payload*
```
// Not my real number
{'phone': '+447633151413'}
```

*Response*
```
{"status":400,"message":"OTP not sent"}
```

Will see if I can capture the request that's being sent from my phone when I log into the app at some point. Maybe the endpoint is different for Europe.

**Comment by Eloise-es** [comment]
Same here, doesn't work with my UK number (+44) but does work with Dutch one (+31). 
I wondered if it's because UK numbers are one digit longer, but I don't think that would explain why @TimHuerzeler's Swiss number doesn't work.

**Comment by LinaAlf** [comment]
> Same here, doesn't work with my UK number (+44) but does work with Dutch one (+31). 
> 
> I wondered if it's because UK numbers are one digit longer, but I don't think that would explain why @TimHuerzeler's Swiss number doesn't work.

Yeah ur right, that wouldn't explain it. Swiss numbers are as follows +41 (0)12 345 67 88
Sometimes you need the zero sometimes you don't, tried worry don't worry, for national calls for instance you drop the +41 and use the 0

**Comment by LinaAlf** [comment]
any update on this? @theOneAndOnlyOne

**Comment by mcraine** [comment]
Also having the same issue with my UK number.

**Comment by dafahan** [comment]
facing the same issue +62 (Indonesia)
